app/models/target.rb

```diff
@@ -27,4 +27,11 @@ class Target < ApplicationRecord
 
   validates :title, :longitude, :latitude, :radius, presence: true
   validates :radius, numericality: { only_integer: true }
+  validate :number_of_user_targets
+
+  private
+
+  def number_of_user_targets
+    errors.add(:user, 'exceeded number of targets limit') if user&.can_add_more_targets?
```

Comment:
I believe that `can_add_more_targets?` as the method name is kind of misleading in this case. I'd expect this method to return true if the user can add more targets, but does the opposite.

What about something like `targets_limit_exceeded?`